.ci/setup_env_ubuntu.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -e
```

Comment:
While testing this PR found that the `-e` didn't had any effect when we call the script using:
 `bash -f ${cidir}/setup_env_ubuntu.sh"` , when errors appeared, the setup continued without failing. 
So I would recommend that we explicitly add  `set -e` to every script.

Reply:
One thing I've considered doing to the metrics scripts is invoking all (sub)scripts with `bash -$- ....`, which then passes on the parent script settings to the child.  That way if you set, say, `-x` on the top level script for debug, it will trickle through to all sub-scripts. Maybe that will work well for your `-e` situation ?